### ISSUE TITLE
Crashes submitting a run after setting 'players' value

### ISSUE THREAD

**Issue by Blantas** [speedruncom/api#78]
Server crashes and doesn't return any response instead of error message "you don't have permissions to set players" while trying to submit a run for specific player with regular user api key.

**Comment by mylogon341** [comment]
It looks like you’re not authenticating with the user your submitting as properly

**Comment by Blantas** [comment]
Yes, but it would be nice if server could return such reason instead of just crashing without any response.

**Comment by mylogon341** [comment]
Is it returning a http status code?

**Comment by Blantas** [comment]
> Is it returning a http status code?
> […](#)
> On 18 Sep 2018, at 6:29 pm, Blantas ***@***.***> wrote: Yes, but it would be nice if server could return such reason instead of just crashing without any response. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or mute the thread.

Exactly. 500.

**Comment by mylogon341** [comment]
Ah ok. Good luck getting that fixed. This api and repo is as dead as it gets. Shame really. They don’t even appear to be considering pull requests, let alone fixing actual bugs.

**Comment by Blantas** [comment]
Well, it's not really a big deal. You either:
a) become a mod so you get permission to submit for others
b) submit others' runs as your own and let mods to edit run

But yea. The whole situation is kinda sad.
